src/tools/perf/lib/libperf.c

```diff
@@ -975,6 +975,7 @@ static void ucp_perf_test_err_handler(void *arg, ucp_ep_h ep,
 {
     ucs_error("error handler called with status %d (%s)\n", status,
               ucs_status_string(status));
+    exit(EXIT_FAILURE);
```

Comment:
as this is perftest specific, suggesting to make sure to reliably exit, rather to flush some ep with ongoing error